certified-connectors/WithoutWire Inventory Platform/apiDefinition.swagger.json

```diff
@@ -1116,6 +1124,4463 @@
+                              },
+                              "QuantityDamaged": {
+                                "type": "number",
+                                "format": "float",
+                                "description": "Represents the quantity damaged on this receipt."
+                              },
+                              "CoreValue": {
+                                "type": "string",
+                                "description": "A string value representing the lot number/serial/expiration date of the item that was picked."
+                              },
+                              "DateReceived": {
+                                "type": "string",
+                                "description": "Date Received"
+                              },
+                              "Identity": {
+                                "type": "integer",
+                                "format": "int32",
+                                "description": "An integer value to uniquely identify."
+                              },
+                              "Status": {
+                                "type": "string",
+                                "description": "Order status name"
+                              },
+                              "LineNumber": {
+                                "type": "integer",
+                                "format": "int32",
+                                "description": "An integer value to uniquely identify the line on the order."
+                              }
+                            }
+                          },
+                          "description": "Receipts"
+                        },
+                        "AllocationSetName": {
+                          "type": "string",
+                          "description": "The allocation set that the associated inventory record is assigned to.",
+                          "title": "Allocation Set Name"
+                        }
+                      }
+                    },
+                    "description": "A collection of Order Line Object to be associated with an order. Receipts are found within the order line object."
+                  }
+                }
+              }
+            }
+          }
+        },
+        "operationId": "GetInboundRequest",
+        "description": "This method allows the exporting of purchase orders by a combination of filters. These filters include begin and end date, receiving status code, item number, line receipt status, purchase order number and purchase order type. The Purchase Order Export Object returns a list of completed order information.",
+        "summary": "Get Inbound Request",
+        "parameters": [
+          {
+            "name": "beginDate",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "\u0421reation date of the purchase order",
```

Comment:
@PavloNikulinEnavate Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.